pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -52,6 +52,14 @@ type BrokerSpec struct {
 	// AuthInfo contains the data that the service catalog should use to authenticate
 	// with the Broker.
 	AuthInfo *BrokerAuthInfo `json:"authInfo,omitempty"`
+
+	// InsecureSkipTLSVerify disables TLS certificate verification when communicating with this Broker.
+	// This is strongly discouraged.  You should use the CABundle instead.
+	// +optional
+	InsecureSkipTLSVerify bool `json:"insecureSkipTLSVerify,omitempty"`
```

Comment:
I think there should be a validation that ensures that you cannot set `InsecureSkipTLSVerify` and `CABundle` at the same time.